_includes/k8s/bestpractices.md

```diff
@@ -0,0 +1,86 @@
+
+## StorageOS Pod placement
+
+StorageOS must run on all nodes that will contribute storage capacity to
+the cluster or that will host Pods which use StorageOS volumes. For production
+environments, it is recommended not to place StorageOS Pods on Master nodes.
+
+StorageOS is deployed with a DaemonSet controller, and therefore tolerates the
+standard unschedulable taint placed on master nodes, and cordoned nodes (see
```

Comment:
I thought that the Daemonset would obey the standard master taint?

Reply:
Nope, it explicitly does not honour the taint to ensure that DaemonSets can run everywhere.